### PR TITLE
Driver verification

### PR DESCRIPTION
**Pull request by Aram-Abovyan**
basic functionality to verify drivers

### CODE REVIEW

File: GG_clone_backend/app/controllers/api/v1/unverified_drivers_controller.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class Api::V1::UnverifiedDriversController < ApplicationController
+  def index
+    render json: Driver.where(is_verified_by_admin: false)
+  end
+
+  def verify
+    driver = Driver.find(params[:id])
+    driver.update_column(:is_verified_by_admin, true)
```
Comment (vnersisyan-vineti) [inline GG_clone_backend/app/controllers/api/v1/unverified_drivers_controller.rb:8]: put empty line here

File: GG_clone_backend/config/routes.rb [hunk 2]
```diff
@@ -15,6 +15,11 @@
       post '/coordinates/trip' => 'coordinates#reservation'
       post '/coordinates/confirm' => 'coordinates#confirm_reservation'
       post '/coordinates/driverAssigned' => 'coordinates#assigned_reservation'
+
```
Comment (vnersisyan-vineti) [inline GG_clone_backend/config/routes.rb:18]: remove comment and empty lines

File: GG_clone_backend/db/seeds.rb [hunk 3]
```diff
@@ -5,25 +5,63 @@
...
+  passenger = Passenger.new(
+    first_name: Faker::Name.first_name,
+    last_name: Faker::Name.last_name,
+    phone_number: "05512#{Faker::PhoneNumber.extension}",
+    email: Faker::Internet.email,
+    password: '123456789'
+  )
+
+  passenger.save!
+end
```
Comment (vnersisyan-vineti) [inline GG_clone_backend/db/seeds.rb:67]: do not forget to put empty line at the end of file

File: GG_clone_backend/db/seeds.rb [hunk 4]
```diff
@@ -5,25 +5,63 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
-Passenger.create({ first_name: 'Hakob', last_name: 'Arshakyan', phone_number: '+37444556699',
-                   email: 'harshakayan@mail.com', password_digest: Passenger.digest('Passenger1') })
-Passenger.create({ first_name: 'Levon', last_name: 'Hakobyan', phone_number: '+37444121232',
-                   email: 'lakobyan777thug@mail.com', password_digest: Passenger.digest('Passenger2') })
-Passenger.create({ first_name: 'Gegham', last_name: 'Antonyan', phone_number: '+37455656522',
-                   email: 'gexamo@mail.com', password_digest: Passenger.digest('Passenger3') })
+# Passenger.create({ first_name: 'Hakob', last_name: 'Arshakyan', phone_number: '+37444556699',
```
Comment (vnersisyan-vineti) [inline GG_clone_backend/db/seeds.rb:8]: if we're going to deprecate this part talk with @iwazowsky and if he's ok with faker approach remove commented section

File: src/components/Admin/verification/UnverifiedDriver.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export default function UnverifiedDriver({ driver }) {
+  const { id, first_name, last_name } = driver;
```
Comment (vnersisyan-vineti) [inline src/components/Admin/verification/UnverifiedDriver.js:4]: destruct these props immediately in argument section

File: yarn.lock [hunk 6]
```diff
@@ -2838,6 +2838,13 @@ binary-extensions@^2.0.0:
   resolved "https://registry.npmjs.org/binary-extensions/-/binary-extensions-2.2.0.tgz"
   integrity sha512-jDctJ/IVQbZoJykoeHbhXpOlNBqGNcwXJKJog42E5HDPUwQTSdjCHdihjj0DlnheQ7blbT6dHOafNAiS8ooQKA==
 
+bindings@^1.5.0:
```
Comment (vnersisyan-vineti) [inline yarn.lock:2841]: we've deleted the yarn.lock file, please also delete it in your branch and do package management with `npm`

**Review by vnersisyan-vineti — CHANGES_REQUESTED** [review]
some minor comments, please address them

File: GG_clone_backend/db/seeds.rb (hunk 4 above)
Comment (iwazowsky) [reply GG_clone_backend/db/seeds.rb:8]: I need this seeds for testing purposes,to achieve predictable results, as I use exact same credentials in api and frontend automation, I guess faker approach will not give me predictable data, am I right?

**Pull request closed without merging**